eplaunch/utilities/version.py

```diff
@@ -1,108 +1,78 @@
 import json
 import os
+from typing import Tuple
+
+# TODO: this built in eplaunch/utilities/version module should be removed, workflows should manage this themselves
 
 
 class Version:
 
-    def check_energyplus_version(self, file_path):
-        _, extension = os.path.splitext(file_path)
-        extension = extension.upper()
-        if extension == '.IDF':
-            found, string_version, number_version = self.check_idf_imf_energyplus_version(file_path)
-        elif extension == '.IMF':
-            found, string_version, number_version = self.check_idf_imf_energyplus_version(file_path)
+    @staticmethod
```

Comment:
Cleaned up this version class, but hoping to just remove it sometime soonish.  It is super E+ workflow specific and we don't really need to provide it to other workflows via EPLaunch.